extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/job/Job.java

```diff
@@ -111,8 +112,16 @@ public static void scheduleRange(String thingUID, AstroThingHandler astroHandler
         if (thingNull || astroHandlerNull || rangeNull || channelIdNull) {
             return;
         }
-        scheduleEvent(thingUID, astroHandler, range.getStart(), EVENT_START, channelId);
-        scheduleEvent(thingUID, astroHandler, range.getEnd(), EVENT_END, channelId);
+        
+        //add 1 second to the last scheduled event if both the events comprise same time instant
+        int secondsPadding = 1;
```

Comment:
I agree. I didn't know about the `DateUtils#truncatedEquals` method. It's more concise. will make the necessary modifications. For the constant, it is only used in one place, that's why I thought it's not a good idea to declare a constant at the interface.